### PR TITLE
feature/pagination

### PR DESCRIPTION
**Pull request by NikPaukov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@
 - Refresh access token - post: 127.0.0.1:3001/api/auth/refresh
 - test authorized - get: 127.0.0.1:3001/api/testAuthorized
 - Proxy Link Preview - `http://127.0.0.1:3001/api/link-preview/?url=${url}`
+- Pagination query info: `https://www.npmjs.com/package/mongoose-paginate-v2`, Url example: `127.0.0.1:3001/api/tweets/?limit=1&page=2&query={"content": "bbb"}`
```
Comment (VitaliiDenysiuk) [inline README.md:24]:
```suggestion
- Pagination query info: `https://www.npmjs.com/package/mongoose-paginate-v2`, Url example: `127.0.0.1:3001/api/tweets/?limit=1&page=2&query={"content": "bbb"}`
- Proxy Link Preview - `http://127.0.0.1:3001/api/link-preview/?url=${url}`
```
move this part above previous row. After that it is explanation proxy link preview

**Review by VitaliiDenysiuk — CHANGES_REQUESTED** [review]

File: src/services/bookmarks.service.ts [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
 import {
   addOne,
   deleteById,
   listBookmarks,
 } from "../repositories/bookmarks.repository";
 
-export const getBookmarks = async () => {
-  const bookmarksList: Bookmark[] = await listBookmarks();
+export const getBookmarks = async (
+  data: PaginationParameters<never, never>
```
Comment (VitaliiDenysiuk) [inline src/services/bookmarks.service.ts:10]: Don't use variable data, set right name

File: src/services/tweets.service.ts [hunk 3]
```diff
@@ -21,8 +23,10 @@ export const deleteTweet = async (id: string) => {
   return deleteOne(id);
 };
 
-export const getAllTweets = async () => {
-  return getAll();
+export const getAllTweets = async (
+  data: PaginationParameters<never, never>
```
Comment (VitaliiDenysiuk) [inline src/services/tweets.service.ts:27]: Don't use variable data, set right name

**Review by VitaliiDenysiuk — CHANGES_REQUESTED** [review]

**Review by alexandrtovmach — APPROVED** [review]
I'm merging it, but please fix in a separate PR all @VitaliiDenysyuk suggestions

**Pull request merged**